resources/dicts/patrols/general/border.json

```diff
@@ -2462,8 +2481,27 @@
                     }
 		],
 	    "weight": 10,
-            "art": "gen_bord_newcatkittypet1"
-	}
+            "art": "gen_kittypet_friends_with_dog1"
+	},
+    {
+        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with her to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified expression.",
+                "exp": 5,
+            "new_cat": [
+            ["kittypet", "meeting", "exists"]
+            ],
+            "relationships": [
```

Comment:
Since you used the "exists" tag, there's the possibility for this event to occur with an outsider who has established relationships with the cats on the patrol.  I'd suggest adding a relationship change to reduce trust/respect/like/comfort as well.